spec/factories/goal.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :goal do
+    description { FFaker::Lorem.sentence }
+    deadline { FFaker::Time.date }
+    status { 'setted' }
```

Comment:
```suggestion
    status { :setted }
```
or you can use `sample` to randomly get a key